connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoClient.scala

```diff
@@ -67,33 +71,14 @@ class KustoClient(val clusterAlias: String, val engineKcsb: ConnectionStringBuil
     engineClient.execute(database, generateTableAlterRetentionPolicy(tmpTableName, "001:00:00:00", recoverable = false))
   }
 
-  private var roundRobinIdx = 0
-  private var storageUris: Seq[ContainerAndSas] = Seq.empty
-  private var lastRefresh: DateTime = new DateTime(DateTimeZone.UTC)
-
-  def getNewTempBlobReference: ContainerAndSas = {
-    // Refresh if storageExpiryMinutes have passed since last refresh for this cluster as SAS should be valid for at least 120 minutes
-    if (storageUris.isEmpty ||
-      new Period(new DateTime(DateTimeZone.UTC), lastRefresh).getMinutes > KustoConstants.storageExpiryMinutes) {
 
-      val res = dmClient.execute(generateCreateTmpStorageCommand())
-      val storage = res.getValues.asScala.map(row => {
-        val parts = row.get(0).split('?'); ContainerAndSas(parts(0), '?' + parts(1))
-      })
 
-      if (storage.isEmpty) {
-        KDSU.reportExceptionAndThrow(myName, new RuntimeException("Failed to allocate temporary storage"), "writing to Kusto", clusterAlias)
-      }
+  def getTempBlobInfoForIngestion: ContainerAndSas = {
```

Comment:
why is it called blob info?

Reply:
its just a url and sas -> should i just drop the `info`?

Reply:
yes